### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by mapedraza**
Move completed features

### CODE REVIEW

**Comment by fgalan** [comment]
> This section has been last updated in August 2023

should be changed to August 2024

File: doc/roadmap.md [hunk 1]
```diff
@@ -37,7 +34,7 @@ product:
 The following list of features are planned to be addressed in the medium term, typically within the subsequent
 release(s) generated in the next 9 months after the next planned release:
 
--   Allow to add metadata to attributes from measures ([#1453](https://github.com/telefonicaid/iotagent-node-lib/issues/1453))
+
```
Comment (mapedraza) [inline doc/roadmap.md:37]:
```suggestion
```

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**